### PR TITLE
Workflow for labeled iOS PRs in 5.x branch

### PR DESCRIPTION
**Pull request by asenyaev**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Review by fengyuentau — APPROVED** [review]
👍

**Comment by asenyaev** [comment]
Fixed parsing a label `platform: ios/osx`.

File: .github/workflows/PR-5.x.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: PR:5.x
 
 on:
   pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
```
Comment (alalek) [inline .github/workflows/PR-5.x.yaml:5]: > labeled, unlabeled

Please don't use this. It retriggers ALL pipelines if any label is updated. It is very inconvenient.

Proposed policy is re-triggering job manually by maintainers after adding necessary label.

Comment (asenyaev) [reply .github/workflows/PR-5.x.yaml:5]: Got your idea, removed this line.

**Comment by asenyaev** [comment]
@fengyuentau, yes, I see. I will add stuck tests in gtest filter.

**Pull request merged**